docs/install.rst

```diff
@@ -40,6 +40,12 @@ Portable                             See the `Windows portable version`_ section
 
 .. _Installing Winget packages: https://docs.microsoft.com/en-us/windows/package-manager/
 
+.. note::
+
+    As of Streamlink version 2.0.0 this software will no longer operate on Windows 7 or Windows Vista.
```

Comment:
technically it is just the installer, not Streamlink

Reply:
the version installed by the installer ;)